src/utils/helpers.test.js

```diff
@@ -56,3 +69,123 @@ describe('getUKFormattedEuros', () => {
     expect(result).toBe('€123.45');
   });
 });
+
+describe('getCurrentMonth', () => {
+  afterEach(() => {
+    global.Date.now = realDateNow;
+  });
+
+  test('if the current date is 2022-06-01, it should return "June 2022"', () => {
+    const date = '2022-06-01';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'June 2022';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
+  });
+  test('if the current date is 2024-02-29, it should return "February 2024"', () => {
+    const date = '2024-02-29';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'February 2024';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
+  });
+  test('if the current date is 2050-12-31, it should return "December 2050"', () => {
+    const date = '2050-12-31';
+    global.Date.now = getDateNowStub(date);
+
+    const expected = 'December 2050';
+    const actual = getCurrentMonth();
+
+    expect(actual).toBe(expected);
+  });
+});
+
+describe('getUniqueMonths', () => {
+  afterAll(() => {
+    global.Date.now = realDateNow;
+  });
+
+  it('should throw an error if no argument is passed', () => {
+    function passZeroArgs() {
+      getUniqueMonths();
+    }
+    expect(passZeroArgs).toThrowError('undefined is not an array');
+  });
+  it('should throw an error if passed a value that is not an array', () => {
+    function passString() {
+      getUniqueMonths('any string');
+    }
+    function passObject() {
+      getUniqueMonths({});
+    }
+    expect(passString).toThrowError('any string is not an array');
+    expect(passObject).toThrowError('[object Object] is not an array');
+  });
```

Comment:
also these two tests we could group to something like "should throw errors for invalid arguments".

Reply:
I personally feel that keeping these separate communicates the different cases/requirements better. It feels more explicit to me, but maybe I'm making a distinction where one doesn't exist.

Reply:
sure, this is just a suggestion. it is hard to weigh "amount of code" against "communicate as specifically as we can". as always: your call.